src/sidebar/components/Annotation/AnnotationShareControl.js

```diff
@@ -188,7 +188,7 @@ function AnnotationShareControl({
           {showShareLinks && <ShareLinks shareURI={shareUri} />}
           <MenuArrow
             direction="down"
-            classes="bottom-[-12px] right-1 touch:right-[9px]"
+            classes="bottom-[-9px] right-1 touch:right-[9px]"
```

Comment:
The icon(s) are slightly different than before and required adjustments to position. The result should be indistinguishable visually from before.